tools/utils.py

```diff
@@ -10,6 +10,7 @@
 import hashlib
 import json
 import os
+from xmlrpc.client import Boolean
```

Comment:
What is this for?!

Reply:
something my editor has added without me recognizing. I remove this.